src/atk.mk

```diff
@@ -19,8 +19,7 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD
-    cd '$(1)' && ./configure \
-        $(MXE_CONFIGURE_OPTS)
-    $(MAKE) -C '$(1)' -j '$(JOBS)' $(MXE_DISABLE_CRUFT) SUBDIRS='atk po' SHELL=bash
-    $(MAKE) -C '$(1)' -j 1 install $(MXE_DISABLE_CRUFT) SUBDIRS='atk po'
+    '$(MXE_MESON_WRAPPER)' $(MXE_MESON_OPTS) -Dintrospection=false '$(BUILD_DIR)' '$(SOURCE_DIR)' && \
+    '$(MXE_NINJA)' -C '$(BUILD_DIR)' -j '$(JOBS)' && \
+    '$(MXE_NINJA)' -C '$(BUILD_DIR)' -j '$(JOBS)' install
```

Comment:
why `&&`? `make` will stop for any logical line failure, something like this is preferrable:
```
    '$(MXE_MESON_WRAPPER)' $(MXE_MESON_OPTS) \
        -Dintrospection=false \
        '$(BUILD_DIR)' '$(SOURCE_DIR)'
     '$(MXE_NINJA)' -C '$(BUILD_DIR)' -j '$(JOBS)'
     '$(MXE_NINJA)' -C '$(BUILD_DIR)' -j '$(JOBS)' install
```

Reply:
Because.. these are 3 different commands, right? You know just append them all together??

Reply:
Cleaned-up on all the makefiles now.